plugin/pkg/admission/priority/admission.go

```diff
@@ -51,8 +52,9 @@ func Register(plugins *admission.Plugins) {
 // priorityPlugin is an implementation of admission.Interface.
 type priorityPlugin struct {
 	*admission.Handler
-	client kubernetes.Interface
-	lister schedulingv1listers.PriorityClassLister
+	client   kubernetes.Interface
+	lister   schedulingv1listers.PriorityClassLister
+	features featuregate.FeatureGate
```

Comment:
no. Use the bools for the things you want to check and assign them.

Reply:
Looking at this comment https://github.com/kubernetes/kubernetes/pull/76310#discussion_r273732346, I made featuregates part of plugin, we don't assign here rather check for those values. If we set it as bool I cannot think of cleaner way to get status for all the featuregates we need inside the admission plugin.